PendingReleaseNotes

```diff
@@ -48,6 +48,9 @@
   until we implement a limit on the number of PGs default pools should consume,
   in combination with the 'scale-down' profile.
 
+* A health warning will now be reported if the ``require-osd-release`` flag is not
+  set to the appropriate release after a cluster upgrade.
+
```

Comment:
Could you please add a new section for 16.2.8 and add this under it?

Reply:
Done.